src/nvim/mouse.c

```diff
@@ -376,6 +406,7 @@ bool do_mouse(oparg_T *oap, int c, int dir, int count, bool fixindent)
     return false;
   }
 
+  static bool in_tab_line = false;   // mouse clicked in tab line
```

Comment:
Having a static variable appear randomly in the middle of a function is strange. Move it back to the top.

Reply:
I tend to agree, though conversely reading these at the top of such a large function tells nothing about where they're actually used.

Reply:
Yes, but these two variables are used in multiple places that are quite far apart :)